docs/getting-started-guides/clc.md

```diff
@@ -87,15 +97,13 @@ guide to install the requirements and install the script.
   source credentials.sh
```

Comment:
`pip install -r requirements.txt` didn't work for me. First errors about missing gcc and 2nd compiling openssl libraries. I had to install gcc-devel and openssl-devel on centos 7